docs/scatacseq/README.md

```diff
@@ -253,7 +253,7 @@ Modality of capturing accessible chromatin molecules.
 
 | constraint | value |
 | --- | --- |
-| enum | `SNARE-Seq2-AC`, `scATACseq`, `bulkATACseq`, `snATACseq`, or `sciATACseq` |
+| enum | `SNARE-Seq2-AC`, `scATACseq`, `Bulk ATAC-seq`, `snATACseq`, or `sciATACseq` |
```

Comment:
This came out of a global find and replace... but it's not really checked for consistency. Should this change be reverted... or should other changes be made?